examples/debug-trans-socket/duk_trans_socket_unix.c

```diff
@@ -11,20 +11,20 @@
  * on all platforms.  The default implementation is to use "poll()"
  * but we can switch that by defining "USE_SELECT" within this
  * file.  For example:
- * 
+ *
  * #define USE_SELECT
- * 
+ *
  * If set, instead of leveraging "poll()", this code will leverage
  * "select()".   Discussions on "poll()" vs "select()" can be read
  * about here: https://daniel.haxx.se/docs/poll-vs-select.html
- * 
```

Comment:
Really small nitpicking but it'd be nice if you could rework to match https://github.com/svaarala/duktape/blob/master/doc/code-issues.rst#comment-styles.